### PR TITLE
https://issues.redhat.com/browse/ACM-9832 MSA default change docs

### PR DESCRIPTION
**Pull request by oafischer**
2.10/2.5 only.

### CODE REVIEW

**Comment by zhujian7** [comment]
@oafischer could you also help remove the [first limitation of the managed service account addon](https://github.com/stolostron/rhacm-docs/blob/2.10_stage/clusters/release_notes/known_issues.adoc#the-managed-serviceaccount-does-not-support-ocp-short-311):

> The managed-serviceaccount does not support {ocp-short} 3.11
The managed-serviceaccount add-on crashes if you run it on {ocp-short} 3.11.

Because in 2.10, we did some enhancement and now the msa addon can run on ocp 311.
story: https://issues.redhat.com/browse/ACM-9655

File: clusters/release_notes/whats_new.adoc [hunk 1]
```diff
@@ -15,7 +15,9 @@ Learn more about what is new this release:
  
 Learn about what's new relating to Cluster lifecycle with {mce-short}.
 
-You can now configure the proxy settings for cluster proxy add-ons to allow a managed cluster to communicate with the hub cluster through a HTTP and HTTPS proxy server. See xref:../cluster_lifecycle/cluster_proxy_addon.adoc#cluster-proxy-addon-settings[Configuring proxy settings for cluster proxy add-ons] to learn more.
+- You can now configure the proxy settings for cluster proxy add-ons to allow a managed cluster to communicate with the hub cluster through a HTTP and HTTPS proxy server. See xref:../cluster_lifecycle/cluster_proxy_addon.adoc#cluster-proxy-addon-settings[Configuring proxy settings for cluster proxy add-ons] to learn more.
+
+- The `ManagedServiceAccount` add-on is now enabled by default. See
```
Comment (zhujian7) [inline clusters/release_notes/whats_new.adoc]: There seems to be a link missing after "See", or delete the "See"?

**Comment by zhiweiyin318** [comment]
Please review when you have a chance @joeg-pro .

File: clusters/release_notes/whats_new.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/release_notes/whats_new.adoc]: Fixed!

File: clusters/cluster_lifecycle/addon_managed_service.adoc [hunk 2]
```diff
@@ -1,7 +1,9 @@
 [#managed-serviceaccount-addon]
 = Enabling ManagedServiceAccount add-ons
 
-When you install the {mce-short}, the `ManagedServiceAccount` add-on is disabled by default. This component when enabled allows you to create or delete a service account on a managed cluster. 
+When you install the {mce-short} version 2.5 or later, the `ManagedServiceAccount` add-on is enabled by default. If you upgrade your hub cluster from {mce-short} version 2.4 and did not enable the `ManagedServiceAccount` add-on before upgrading, you have to enable the add-on manually.
```
Comment (joeg-pro) [inline clusters/cluster_lifecycle/addon_managed_service.adoc]: I think we should be explicit in noting the change in behavior.   So, I suggest that after the "When you install..." setence, we add a new one that reads  "This is a change from version 2.4 and earlier where ManagedServiceAccount add-on was disabled by default."

**Review by joeg-pro — CHANGES_REQUESTED** [review]
See 1 comment.

File: clusters/cluster_lifecycle/addon_managed_service.adoc (hunk 2 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/addon_managed_service.adoc]: Updated, thanks!

File: clusters/cluster_lifecycle/addon_managed_service.adoc [hunk 3]
```diff
@@ -1,7 +1,9 @@
 [#managed-serviceaccount-addon]
 = Enabling ManagedServiceAccount add-ons
 
-When you install the {mce-short}, the `ManagedServiceAccount` add-on is disabled by default. This component when enabled allows you to create or delete a service account on a managed cluster. 
+When you install the {mce-short} version 2.5 or later, the `ManagedServiceAccount` add-on is enabled by default. In {mce-short} version 2.4 and earlier, the `ManagedServiceAccount` add-on was disabled by default. If you upgrade your hub cluster from {mce-short} version 2.4 and did not enable the `ManagedServiceAccount` add-on before upgrading, you have to enable the add-on manually.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/addon_managed_service.adoc]: From my perspective, it is unnecessary to mention this statement, and it's past tense: In {mce-short} version 2.4 and earlier, the `ManagedServiceAccount` add-on was disabled by default.

```suggestion
When you install the {mce-short} version 2.5 or later, the `ManagedServiceAccount` add-on is enabled by default. If you upgrade your hub cluster from {mce-short} version 2.4 and did not enable the `ManagedServiceAccount` add-on before upgrading, you must enable the add-on manually.
```

**Review by dockerymick — APPROVED** [review]
Made a couple of suggestions. Other than that, this looks good to me!

File: clusters/cluster_lifecycle/addon_managed_service.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/addon_managed_service.adoc]: Joe wanted us to have that sentence. Not sure how to proceed here.

**Pull request merged**